gdal/doc/source/api/gdalogrin_python.rst

```diff
@@ -0,0 +1,229 @@
+.. _gdalogrin_python:
+
+================================================================================
+GDAL/OGR in Python
+================================================================================
+
+This Python package and extensions are a number of tools for programming and manipulating the `GDAL <http://www.gdal.org/>`__ Geospatial Data Abstraction Library. Actually, it is two libraries -- GDAL for manipulating geospatial raster data and 
+OGR for manipulating geospatial vector data -- but we'll refer to the entire package as the GDAL library for the purposes of this document.
+
+The GDAL project (primarily Howard Butler) maintains SWIG generated Python bindings for GDAL and OGR.
+
+API
+---
+
+Generally speaking the classes and methods mostly match those of the GDAL and OGR C++ classes. An automatically generated API documentation is available on `http://gdal.org/python <http://gdal.org/python>`__.
+It is far from being ideal however : you will essentially get the names of available classes and functions, but few descriptions (except for OGR classes where the OGR C documentation is copied). For more details,
+look at the `GDAL API Tutorial <http://www.gdal.org/gdal_tutorial.html>`__ includes Python examples.
+
+Tutorials
+---------
+
+Chris Garrard has given courses at Utah State University on "Geoprocessing with Python using Open Source GIS" (`http://www.gis.usu.edu/~chrisg/python <http://www.gis.usu.edu/~chrisg/python>`__). There are many slides, examples, test data... and homework ;-) that can
+be greatly helpful for beginners with GDAL/OGR in Python.
+
+Greg Petrie has an OGR related tutorial available at `http://cosmicproject.org/OGR <http://cosmicproject.org/OGR>`__.
+
+A cookbook full of recipes for using the Python GDAL/OGR bindings : `http://pcjericks.github.io/py-gdalogr-cookbook/index.html <http://pcjericks.github.io/py-gdalogr-cookbook/index.html>`__
+
+Examples
+--------
+
+* One example of GDAL/numpy integration is found in the val_repl.py script.

```

Comment:
please update the links in this paragraph to github

Reply:
Fixed